### PR TITLE
Chore/remove x ray

### PR DESCRIPTION
**Pull request by numidiasoft**
## 🐻 What your PR is doing?

Remove the x-ray package from the Bearer Framework

## 📦 Package concerned

- @bearer/cli
- @bearer/functions

## ✅ Checklist

- [ ] Tests were added (if necessary)
- [ ] I used conventional commits

### CODE REVIEW

File: packages/functions/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "@bearer/logger": "^1.7.0",
-    "@bearer/x-ray": "^1.7.0",
```
Comment (tanguyantoine) [inline packages/functions/package.json]: I don't think we need that anymore

File: packages/functions/src/functions/fetch.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import debug from '@bearer/logger'
-import { setupFunctionIdentifiers } from '@bearer/x-ray'
```
Comment (tanguyantoine) [inline packages/functions/src/functions/fetch.ts]: the package is private now, that means that people not having access to it (every people external to Bearer) won't be able to install and bootstrap the bearer toolkit

Comment (numidiasoft) [reply packages/functions/src/functions/fetch.ts]: it still public

File: packages/functions/package.json (hunk 1 above)
Comment (numidiasoft) [reply packages/functions/package.json]: we need to sync about that, in fact, if I remove it from dependencies, how the developer can get the dependency locally, should we add it to the package.json of the function?

Comment (numidiasoft) [reply packages/functions/package.json]: Yes, will be removed

**Pull request merged**